server/routes/smallgroups.js

```diff
@@ -0,0 +1,138 @@
+import moment from 'moment';
+import cloudinary from 'cloudinary';
+import formidable from 'formidable';
+import admin from 'firebase-admin';
+import { promisify } from 'util';
+import { replaceURLsWithLinks } from '../lib';
+
+const sgRef = admin.firestore().collection('smallgroups');
+
+/*
+getSmallGroups gets small groups 
+*/
+export const getSmallGroups = async (req, res) => {
+    try {
+      const msg = await sgRef.doc('men').collection('men_sgs').get();
+      const menSgs = [];
+        if (!msg.empty) {
+          msg.forEach((doc) => {
+            if (doc.exists) {
+              const course = doc.data();
+              menSgs.push(course);
+            }
+          });
+        }
+      const fsg = await sgRef.doc('women').collection('women_sgs').get();
+      const womenSgs = [];
+        if(!fsg.empty){
+            fsg.forEach((doc) => {
+              if(doc.exists) {
+                const course = doc.data();
+                womenSgs.push(course);
+              }
+            })
+        }
+        res.render('smallgroups.ejs', {
+          title: 'Small Groups',
+          menSgs,
+          womenSgs,
+        });
+      } catch (e) {
+        res.status(500).json(e);
+      }
+};
+
+// moment(req.body.date).unix()
```

Comment:
Why is all this commented out? If its work in progress probably put it into another branch :)